src/Shared/Data/Database/Monsters.cs

```diff
@@ -30,6 +30,12 @@ public class MonsterData
 		public int Exp { get; set; }
 		public int JobExp { get; set; }
 
+		public int STR { get; set; }
```

Comment:
We generally try to avoid the upper case naming-scheme unless there's a good reason for it, as PascalCase is more idiomatic to C#. We're also already using `Str`, `Con`, etc. in other places, which makes these stand out a little.